test failure

Gentoo overlay guru CI fail tests with:
```
>>> Test phase: dev-python/SQLAlchemy-Utils-0.37.8
 * python3_8: running distutils-r1_run_phase python_test
python3.8 -m pytest -vv -ra -l -Wdefault --deselect tests/types/test_timezone.py
============================= test session starts ==============================
platform linux -- Python 3.8.12, pytest-6.2.5, py-1.10.0, pluggy-1.0.0 -- /usr/bin/python3.8
cachedir: .pytest_cache
rootdir: /var/tmp/portage/dev-python/SQLAlchemy-Utils-0.37.8/work/SQLAlchemy-Utils-0.37.8
collecting ... collected 865 items / 1 error / 864 selected

==================================== ERRORS ====================================
________________ ERROR collecting tests/types/test_timezone.py _________________
ImportError while importing test module '/var/tmp/portage/dev-python/SQLAlchemy-Utils-0.37.8/work/SQLAlchemy-Utils-0.37.8/tests/types/test_timezone.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python3.8/site-packages/_pytest/python.py:578: in _importtestmodule
    mod = import_path(self.fspath, mode=importmode)
/usr/lib/python3.8/site-packages/_pytest/pathlib.py:524: in import_path
    importlib.import_module(module_name)
/usr/lib/python3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:1014: in _gcd_import
    ???
<frozen importlib._bootstrap>:991: in _find_and_load
    ???
<frozen importlib._bootstrap>:975: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:671: in _load_unlocked
    ???
/usr/lib/python3.8/site-packages/_pytest/assertion/rewrite.py:170: in exec_module
    exec(co, module.__dict__)
tests/types/test_timezone.py:4: in <module>
    from dateutil.zoneinfo import getzoneinfofile_stream, tzfile, ZoneInfoFile
E   ImportError: cannot import name 'getzoneinfofile_stream' from 'dateutil.zoneinfo' (/usr/lib/python3.8/site-packages/dateutil/zoneinfo/__init__.py)
=========================== short test summary info ============================
ERROR tests/types/test_timezone.py
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
=============================== 1 error in 2.27s ===============================
```
See https://bugs.gentoo.org/819021

--- a/tests/types/test_timezone.py
+++ b/tests/types/test_timezone.py
@@ -1,7 +1,7 @@
 import pytest
 import pytz
 import sqlalchemy as sa
-from dateutil.zoneinfo import getzoneinfofile_stream, tzfile, ZoneInfoFile
+from dateutil.zoneinfo import get_zonefile_instance, tzfile
 
 try:
     import zoneinfo
@@ -89,8 +89,7 @@ def test_can_coerce_string_for_pytz_zone(zone):
     assert tzcol._coerce(zone).zone == zone
 
 
-@pytest.mark.parametrize(
-    'zone', ZoneInfoFile(getzoneinfofile_stream()).zones.keys())
+@pytest.mark.parametrize('zone', get_zonefile_instance().zones.keys())
 def test_can_coerce_string_for_dateutil_zone(zone):
     tzcol = TimezoneType(backend='dateutil')
     assert isinstance(tzcol._coerce(zone), tzfile)

